packages/taxdat/R/plot_table_by_admin.R

```diff
@@ -29,23 +29,32 @@ if(nrow(cache[["sf_grid"]]) == prod(dim(pop_layer))){
   ### Use the geo package
   iso_code <- as.character(stringr::str_extract(config, "[A-Z]{3}"))
   admin_level <- as.numeric(admin_level_for_summary_table)
-  if ((iso_code == "ZNZ" | grepl("zanzibar", tolower(config))) & admin_level == 1){
-    boundary_sf <- rgeoboundaries::gb_adm1("TZA")[rgeoboundaries::gb_adm1("TZA")$shapeName %in% 
-      c("Zanzibar South & Central", "Zanzibar North", "Zanzibar Urban/West", "North Pemba", "South Pemba"), ]
-  } else if ((iso_code == "ZNZ" | grepl("zanzibar", tolower(config))) & admin_level == 2){
-    boundary_sf <- rgeoboundaries::gb_adm2("TZA")[rgeoboundaries::gb_adm2("TZA")$shapeName %in% 
-      c("Micheweni", "Wete", "Chake Chake", "Mkoani", 
-        "Kaskazini A", "Kaskazini B", "Mjini", "Magharibi", "Kati", "Kusini"), ]
+  if ((iso_code == "ZNZ" | grepl("zanzibar", tolower(config)))){
+    boundary_sf =sf::st_as_sf(gadm(country="TZA", level=admin_level, path=tempdir()))%>%subset(NAME_1%in%c("Kaskazini Pemba","Kaskazini Unguja","Kusini Pemba","Kusini Unguja"))
+    boundary_sf%>%
+    janitor::clean_names() %>%
+    mutate(country="Zanzibar",

```

Comment:
@QLLZ Can you add the explicit package names in front of the function names in a systematic way?